deeplinkdispatch/src/main/java/com/airbnb/deeplinkdispatch/BaseDeepLinkDelegate.kt

```diff
@@ -304,7 +304,7 @@ open class BaseDeepLinkDelegate @JvmOverloads constructor(
                 if (intent.data == null) {
                     intent.data = sourceIntent.data
                 }
-                intent.putExtras(intentBundle)
+                intent.putExtras(intentBundle.filter { key, _ -> intent.extras?.let { !it.containsKey(key) } ?: true })
```

Comment:
I think this is what you want?
```suggestion
                intent.putExtras(intentBundle.filter { key, _ -> intent.extras?.containsKey(key) != true )
```

Reply:
Yeah, I did have it kinda like that yesterday (before reverting to the ugly way of doing it but I could just not wrap my mind around it. Thank you!